apps/services/schemas.py

```diff
@@ -13,9 +13,73 @@
 )
 
 
-class CreateUserRequestIn(BaseInSchema):
-    service_id: int
+class RequestForHostelAccommodationIn(BaseInSchema):
```

Comment:
It's better to write titles, descriptions and provide a `Field` instance for every field in schemas.

Reply:
ok